modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/model/listener/OrganizationExpandoColumnModelListener.java

```diff
@@ -185,6 +186,13 @@ else if (expandoColumn.getType() == ExpandoColumnConstants.DATE) {
 				locale -> Field.getSortableFieldName(encodedIndexedFieldName),
 				locale -> encodedIndexedFieldName);
 		}
+		else if ((expandoColumn.getType() == ExpandoColumnConstants.SHORT) ||
```

Comment:
Normally It's good to sort conditions alphabetically, if logic is not affected with the change

Reply:
Addressed in #37.